cmd/monaco/integrationtest/v2/scoped_configs_test.go

```diff
@@ -36,16 +43,81 @@ func TestDeployScopedConfigurations(t *testing.T) {
 	envVars := map[string]string{
 		featureflags.MRumProperties().EnvName():         "true",
 		featureflags.DashboardShareSettings().EnvName(): "true",
-		dashboardSharedEnvName:                          "false",
 	}
 
-	RunIntegrationWithCleanupGivenEnvs(t, configFolder, manifest, environment, "ScopedConfigs", envVars, func(fs afero.Fs, _ TestContext) {
+	RunIntegrationWithCleanupGivenEnvs(t, configFolder, manifest, environment, "ScopedConfigs", envVars, func(fs afero.Fs, testContext TestContext) {
 
-		cmd := runner.BuildCli(fs)
-		cmd.SetArgs([]string{"deploy", "--verbose", manifest, "--environment", environment})
-		err := cmd.Execute()
+		// deploy with sharing turned off and assert state
+		t.Setenv(integrationtest.AddSuffix(dashboardSharedEnvName, testContext.suffix), "false")
```

Comment:
nit: I'd just export `setTestEnvVar` from the _utils file.
The suffix addition to env vars is an ugly workaround to the fact that test utils are rewriting all 'name' fields - including the environment variable name for env-var parameters... That can continue to be hidden behind the helper function, which by setting both also allows us to fix this at some point without breaking tests